Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:21-slim
+FROM eclipse-temurin:21.0.5_11-jre-noble
```

Comment:
Is there a reason you chose the ubuntu image here?
[This specific one](https://hub.docker.com/layers/library/eclipse-temurin/8u432-b06-jre-noble/images/sha256-70b116af7384a0248d8f94dafe2a0dffd084af4aea9692d14a4f575489c79868?context=explore) has multiple vulnerabilities in it.

If there is no reason, please use an alpine based image.
```suggestion
FROM eclipse-temurin:21.0.5_11-jre-alpine
```

Reply:
Because of the size ;-) And we are using currently the eclipse-temurin:21.0.5_11-jre-noble in our portal backend with no HIGH vulnerabilities